spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -96,6 +97,8 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
     val rMapped = dfrdd.map {
       partdf ⇒
         try {
+          // check if Rserve is running, if not: start it
+          if(!StartRserve.checkLocalRserve()) throw new RuntimeException("Unable to start Rserve")
```

Comment:
More detailed messages will be put to console inside StartRserve.checkLocalRserve()